Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM python:3.4-onbuild
+# https://github.com/docker-library/official-images/issues/2076
+FROM python:3.4
+
+RUN pip install pipenv --upgrade
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
+COPY . /usr/src/app
```

Comment:
If we put this at the bottom of the file it will be most efficient for layer caching